database/init-tables.sql

```diff
@@ -738,7 +739,16 @@ ALTER TABLE ONLY public."user"
 
 
 --
--- TOC entry 2899 (class 2606 OID 16534)
+-- TOC entry 2896 (class 2606 OID 16712)
+-- Name: user user_primary_email_key; Type: CONSTRAINT; Schema: public; Owner: -
+--
+
+ALTER TABLE ONLY public."user"
+    ADD CONSTRAINT user_primary_email_key UNIQUE (primary_email);
+
+
+--
+-- TOC entry 2901 (class 2606 OID 16540)
```

Comment:
Adds unique constraint to primary email.

Reply:
looks fine to me

Reply:
Ignore the previous issue. I forgot when a local sql is updated in pgadmin it will be done through an `alter` in sql file at the end. So we should be good.